python/lang/security/audit/md5-used-as-password.yaml

```diff
@@ -4,7 +4,8 @@ rules:
   message: >-
     It looks like MD5 is used as a password hash. MD5 is not considered a
     secure password hash because it can be cracked by an attacker in a short
-    amount of time. Use SHA-256 or higher for password hashes (for example: `hashlib.sha256(...)`)
+    amount of time. Use a suitable password hashing function such as scrypt.
+    You can use `hashlib.script`.
```

Comment:
`script` should be `scrypt` here.

Reply:
Thanks a lot :pray:

Reply:
PR: https://github.com/returntocorp/semgrep-rules/pull/1653